lib/slim_fast/compiler.ex

```diff
@@ -9,15 +9,22 @@ defmodule SlimFast.Compiler do
 
   defp render_attribute(_, []), do: ""
   defp render_attribute(_, ""), do: ""
+  defp render_attribute(name, {:eex, opts}) do
+    value = opts[:content]
+    case value do
+      "true"  -> name
+      "false" -> ""
+      "nil"   -> ""
+      _       -> "#{name}=<%=#{value}%>"
```

Comment:
I think it is good enough for now. There is no unquoted attributes support in slim, except you can use inline html for that.